es6 & beyond/ch1.md

```diff
@@ -108,7 +108,7 @@ Se você decidir deixar como está e esperar que todos os navegadores sem as fun
 
 ## Revisando
 
-ES6 (alguns podem tentar chamá-lo de ES2015) é apenas o pouso a partir do momento em que este está sendo escrito, e tem muitas coisas novas que você precisa aprender!
+ES6 (alguns podem tentar chamá-lo de ES2015) está apenas chegando a partir do momento em que este está sendo escrito, e tem muitas coisas novas que você precisa aprender!
```

Comment:
A frase original é `[...] is just landing as of the time of this writing`.
O que acha de mudar esse trecho?
```suggestion
ES6 (alguns podem tentar chamá-lo de ES2015) está apenas chegando no momento em que este livro está sendo escrito, e tem muitas coisas novas que você precisa aprender!
```